test/cases/040_packages/032_bcc/test.yml

```diff
@@ -0,0 +1,29 @@
+kernel:
+  image: linuxkit/kernel:4.14.54
+  cmdline: "console=ttyS0 console=ttyAMA0"
+init:
+  - linuxkit/init:v0.5
+  - linuxkit/runc:v0.5
+  - linuxkit/containerd:v0.5
+  - linuxkit/ca-certificates:v0.5
+  - linuxkit/kernel-bcc:4.14.54
+services:
+# A service which generates logs of log messages
+  - name: check-bcc
```

Comment:
this should be implementable as a `onboot` as most other tests. The `logwrite` test you copied this from need to run this as a service